### PR TITLE
Cleanup ppmc md

### PR DESCRIPTION
**Pull request by ulysses-you**

### CODE REVIEW

**Comment by ulysses-you** [comment]
cc @yaooqinn

File: src/en/new_ppmc_member_process.md [hunk 1]
```diff
@@ -24,14 +24,15 @@ menu:
...
-- [The process of new PPMC member](new_ppmc_member_process.html#the-process-of-new-pmc-member)
-    - [Call a vote in mailing `private@kyuubi.apache.org`](new_ppmc_member_process.html#call-a-vote-in-mailing-privatekyuubiapacheorg)
-    - [Close the vote](new_ppmc_member_process.html#close-the-vote)
-    - [Send message to IPMC](new_ppmc_member_process.html#send-message-to-ipmc)
-    - [If the result is positive, invite the new PPMC member](new_ppmc_member_process.html#if-the-result-is-positive-invite-the-new-ppmc-member)
-    - [If accept, then: Accept the PPMC member](new_ppmc_member_process.html#if-accept-then-accept-the-ppmc-member)
-    - [Notify the PPMC member of completion](new_ppmc_member_process.html#notify-the-ppmc-member-of-completion)
-    - [Announce the new PPMC member](new_ppmc_member_process.html#announce-the-new-ppmc-member)
+- The process of new PPMC member
+    - Discuss in mailing `private@kyuubi.apache.org`
```
Comment (yaooqinn) [inline src/en/new_ppmc_member_process.md:28]: Discuss in the private mailing list

File: src/en/new_ppmc_member_process.md [hunk 2]
```diff
@@ -24,14 +24,15 @@ menu:
...
+    - Call a vote in mailing `private@kyuubi.apache.org`
```
Comment (yaooqinn) [inline src/en/new_ppmc_member_process.md:29]: Call a vote in the private mailing list

File: src/en/new_ppmc_member_process.md [hunk 3]
```diff
@@ -24,14 +24,15 @@ menu:
...
+    - Close the vote
+    - Send message to IPMC
```
Comment (yaooqinn) [inline src/en/new_ppmc_member_process.md:31]: Send a NOTICE to IPMC

File: src/en/new_ppmc_member_process.md [hunk 4]
```diff
@@ -24,14 +24,15 @@ menu:
...
+    - If the result is positive, invite the new PPMC member
+    - If accept, then: Accept the PPMC member
```
Comment (yaooqinn) [inline src/en/new_ppmc_member_process.md:33]: If the invitation is accepted, then accept the PPMC member

**Comment by yaooqinn** [comment]
please fix the grammar issue to make it less Chinglish

**Review by yaooqinn — APPROVED** [review]

**Pull request merged**